tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -5798,6 +5811,16 @@ static TableParameter PackUnpackOpParameters[] = {
     { L"Validation.Input", TableParameter::UINT32_TABLE, true },
 };
 
+static TableParameter IsNormalParameters[] = {
```

Comment:
This is unused.  I imagine copied/adapted from the UnaryFloatOpTest.  This test isn't data driven like UnaryFloatOpTest is.